apps/condo/domains/common/components/GraphQlSearchInput.tsx

```diff
@@ -65,6 +65,11 @@ export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
     function handleSelect (value, option) {
         setSelected(option.children)
 
+        if (props.mode === 'multiple') {
```

Comment:
What is this fix about? Please describe your solution

Reply:
In shareTIcketModal, we have a multiselect for user choice, in which we need to clear the search field after selecting the user with whom we want to share the ticket.